### ISSUE TITLE
Salvage furniture from empty offices

### ISSUE THREAD

**Issue by huffeec** [DoESLiverpool/somebody-should#631]
There are desks and chairs in the empty rooms on floors 3, 5 and 7.
With @helicalbytes we salvaged 7 wheeled office chairs and 2 2m desks and put them in the staging room. 
There are lots more that may be useful.
Before we collect them it would be useful to measure the desks and make sure they have a use/space in the new plan.
![img_20171219_121909826](https://user-images.githubusercontent.com/4748179/34159866-af2cdacc-e4c2-11e7-8a8a-bbb798a20dcd.jpg)

Please make your issue:
* Specific
* Measurable
* Achievable
* Responsible
* Time-constrained

You can delete this template text.

**Comment by johnmckerrell** [comment]
![2017-12-19 14 55 34](https://user-images.githubusercontent.com/27243/34483959-952eae8c-efb9-11e7-9ade-5764a3961134.jpg)
![2017-12-19 14 54 11](https://user-images.githubusercontent.com/27243/34483960-95516ada-efb9-11e7-8e5d-5ae39e038d1a.jpg)

These were on the 7th floor and look interesting although we'd probably need to confirm about the dining table and chairs. I suppose they probably wouldn't be very versatile (or comfy) so might look good in a board room but would be a pita if we needed a more open space.

**Comment by amcewen** [comment]
As mentioned in #644, a couple of metal cupboards would be handy for the workshop areas.  Checking out the empty rooms on the third floor, there are two metal cupboards there.  One similar to the one we've got in the corner of the main room, in the old Kaos kids rooms.  And one with roller-doors in the old language school rooms.

I don't know how fireproof the one with roller-doors would be (it'd depend what the doors are made of), but I wondered if it'd be any use to house the 3D printers?  That'd protect the surrounding area in case they caught fire (unlikely, but they are one of the unattended risks); allow them to be stacked more easily; and the roller-doors wouldn't get in the way if left open...  (I've not measured them to check if the printers would fit - that might be a deal-breaker for them :-)

If anyone wants to pull them up to the staging room sometime that'd be great.

**Comment by amcewen** [comment]
If we want anything from the Kaos rooms downstairs we should get it sooner rather than later as those rooms are getting reused by people being shuffled round the building in a week or two.

**Comment by johnmckerrell** [comment]
There's a big trolley/trailer thing by the lift on our floor. Not sure who's it is but I'm sure they wouldn't mind us borrowing it if we're looking to move anything. (Not actually sure it fits in the lift) We should get the fire doors sometime too and ideally measure them.

**Comment by johnmckerrell** [comment]
There's some big white desks in the room above ours which could be ideal for the workshop. They're double sized (i.e. like two desks stuck together on the long side) so will need taking apart before moving to get them out of the door.

Hate saying this when I'm not doing it but we really should move the doors and these desks as soon as we can, the key to the office above us should be in the main room. I'll try to move some stuff on Monday assuming it's not all been done!

**Comment by amcewen** [comment]
After Maker Day on Saturday, @helicalbytes, Darren, @goatchurchprime and I did a load of dismantling and moving of desks from the fifth floor.  There are still four white desks (two lots of double-desks) to dismantle, and some of the dismantled desks to bring down.

There's also a pair of glass hatch doors (and runners) and some Ethernet switches on a table to bring down.  Plus a server cabinet which might work better than a metal cupboard for holding the Ultimakers (having measured them - they'll fit in the deeper cabinet, but probably not a cupboard unless we move the spools from the back to somewhere else).

There are a load of computers there too, but I've spun that out as a separate issue (#657) as it needs a bit more work than just "carry things down".

**Comment by amcewen** [comment]
More of the stuff has been moved down now and is all in the staging room (inc. the fire doors).

Still to be moved down from the 5th floor are four more white/chrome desks (they need dismantling first - that requires a large philips screwdriver and an allen key), and the server cabinet (that needs all the cables coming out of it to be cut first before it'll move).  Maybe also some of the chairs if they're decent?

**Comment by amcewen** [comment]
Andy G and I dismantled the rest of the desks on the 5th floor, and moved them and the server cabinet down into the staging room.  There's nothing else worth taking from there, apart from possibly some of the chairs and the 2-port network points on the wall (if they'll be useful and anyone wants to get to that level of disassembly)

**Comment by amcewen** [comment]
The fire doors we've salvaged are of various sizes.  I discussed this with Dave just now (he was checking the sizes), and we agreed not to use the 30" wide doors (there are two of those in the ones we've salvaged, IIRC) because wider ones will be better for getting things (equipment, wheelchairs...) through.  @Robotorium couldn't get into the kitchen here, for example, which has a 29" door.  Dave is going to see if there are any more decent doors we can salvage instead, and aim for 33" or 34" ones.
